test/mmdb_util.rb

```diff
@@ -0,0 +1,24 @@
+class MMDBUtil # :nodoc:
+  def self.make_metadata_map(record_size)
+    # Map
+    "\xe9".b +
```

Comment:
Just wondering if you want to use `<<` here. https://github.com/rubocop-hq/ruby-style-guide#concat-strings

Reply:
Normally I'd agree, but this is just a test thing, so it does not seem important.